libs/api/books/src/lib/books.service.ts

```diff
@@ -15,20 +15,25 @@ export class BooksService {
     return this.http
       .get(`https://www.googleapis.com/books/v1/volumes?q=${term}`)
       .pipe(
-        map(resp => {
-          return resp.data.items.map(item => {
-            return {
-              id: item.id,
-              title: item.volumeInfo?.title,
-              authors: item.volumeInfo?.authors || [],
-              description: item.searchInfo?.textSnippet,
-              publisher: item.volumeInfo?.publisher,
-              publishedDate: item.volumeInfo?.publishedDate
-                ? new Date(item.volumeInfo?.publishedDate).toISOString()
-                : undefined,
-              coverUrl: item.volumeInfo?.imageLinks?.thumbnail
-            };
-          });
+        map((resp) => {
+          if (resp.data.totalItems > 0) {
+            console.log('SOOOOO OH AHHHHHHH :: ', resp);
```

Comment:
🤨

Reply:
Welp, I was sure I deleted that one. I will remove it with the changes.